pkg/serverless/invocationlifecycle/lifecycle.go

```diff
@@ -46,6 +54,13 @@ func (lp *LifecycleProcessor) OnInvokeEnd(endDetails *InvocationEndDetails) {
 		endExecutionSpan(lp.ProcessTrace, endDetails.RequestID, endDetails.EndTime, endDetails.IsError)
 	}
 
+	if inferredspan.IsExpected() {
+		log.Debug("[lifecycle] Attempting to complete the inferred span")
+		inferredspan.CompleteInferredSpan(
```

Comment:
I think we only want to do this if the Lambda Library is NOT detected -- otherwise we might create a duplicate inferred span (if the Lambda Library is used as well)